confidant/services/jwkmanager.py

```diff
@@ -22,15 +25,102 @@
 CA_SCHEMA = {
     'crt': {'type': 'string', 'required': True},
     'key': {'type': 'string', 'required': True},
-    'passphrase': {'type': 'string', 'required': True},
+    'passphrase': {'type': 'string', 'required': True, 'nullable': True},
```

Comment:
What do we use this for?

Reply:
https://github.com/lyft/confidant/blob/96c5f5bd907efe03f2885c2e8eb42c28bf0f1f4f/confidant/services/jwkmanager.py#L43

It fails if `passphrase` is None, which it currently is in our unit tests https://github.com/lyft/confidant/blob/96c5f5bd907efe03f2885c2e8eb42c28bf0f1f4f/tests/conftest.py#L159

It's failing after I updated our unit tests to actually hit this path.